macros/upload_artifacts_v2.sql

```diff
@@ -42,13 +43,46 @@
             args:target::string as target,
             metadata,
             args
-        from raw_data;
+        from raw_data
+    ) as new_data
+    on old_data.artifact_run_id = new_data.artifact_run_id and old_data.artifact_generated_at = new_data.artifact_generated_at
```

Comment:
```suggestion
    on old_data.artifact_run_id = new_data.artifact_run_id
```

I don't think the existing is bad - but again just for consistency with the dedupe script we should just be able to use artifact_run_id here, right?

Reply:
In line with #103 and #94 - we should use `command_invocation_id` (+/- `node_id`) for these merges.